src/client/sdp/home/home.component.html

```diff
@@ -131,7 +131,7 @@
       </a>
     </div>
     <div class="ui-g-12 ui-md-3 ui-lg-2">
-      <a  href = "#!"  (click)="search('','Physics Neutron','')" style="text-decoration: none">
+      <a  href = "#!"  (click)="search('','Physics,Neutron','')" style="text-decoration: none">
```

Comment:
This is for Featured Research blocks, we need to pass these parameters for quick search